docs/conventions.md

```diff
@@ -72,7 +72,4 @@ If you have customized `RoIHead`, you can refer to the above method to deal with
 
 In MMDetection, we have supported COCO Panoptic dataset. We clarify a few conventions about the implementation of `CocoPanopticDataset` here.
 
-1. In the `gt_semantic_seg`, different from the default setting of MMDetection, the label `0` stands for `VOID` label and the category labels start from `1`.
```

Comment:
These contents should be kept and indicate their version. So that users using versions before this PR will not be confused.